gen3_util/access/submitter.py

```diff
@@ -0,0 +1,25 @@
+
+from gen3.auth import Gen3Auth
+from gen3.submission import Gen3Submission
+
+from gen3_util import Config
+from gen3_util.config import ensure_auth
+
+
+def ensure_program_project(config: Config, project_id: str, auth: Gen3Auth = None) -> str:
+    """Ensure program and project exist in sheepdog.
+    """
+    if not auth:
+        auth = ensure_auth(profile=config.gen3.profile)
+    program, project = project_id.split('-')
+    submission = Gen3Submission(auth)
```

Comment:
Would be nice to have a test for this but since it accesses sheepdog, can see why no test

Reply:
Yes, see below https://github.com/ACED-IDP/gen3_util/pull/31/files#r1434678711